moon/controller_round3.py

```diff
@@ -111,6 +110,8 @@ def __init__(self, config, bus):
         super().__init__(config, bus)
         bus.register("desired_movement")
 
+        self.default_effort_level = 600
```

Comment:
500 should represent originally tested performance (effort=40)
```suggestion
        self.default_effort_level = 500
```